addons/l10n_es/data/template/account.account-es_pymes.csv

```diff
@@ -1,37 +1,45 @@
-"id","name","name@es","code","account_type"
-"account_pymes_100","Share capital","Capital social","100","liability_non_current"
-"account_pymes_1030",Uncalled capital,"Socios por desembolsos no exigidos, capital social","1030","liability_non_current"
-"account_pymes_1034","Uncalled capital pending registration","Socios por desembolsos no exigidos, capital pendiente de inscripción","1034","liability_non_current"
-"account_pymes_1040","Uncalled non-monetary contributions, capital","Socios por aportaciones no dinerarias pendientes, capital social","1040","liability_non_current"
-"account_pymes_1044","Uncalled non-monetary contributions, capital pending registration","Socios por aportaciones no dinerarias pendientes, capital pendiente de inscripción","1044","liability_non_current"
-"account_pymes_120","Retained earnings","Remanente","120","liability_non_current"
-"account_pymes_121","Prior period's losses","Resultados negativos de ejercicios anteriores","121","liability_non_current"
-"account_pymes_129","Profit/loss for the period","Resultado del ejercicio","129","equity_unaffected"
```

Comment:
We recently changed this to equity_unaffected, so it would be nice to keep that change.

Reply:
What do you mean? It's still `equity_unaffected`. I just added the `reconcile` column here.

Reply:
indeed, looking at the wrong line